Common/src/containers/CFileReaderLUT.cpp

```diff
@@ -59,33 +59,77 @@ void CFileReaderLUT::ReadRawLUT(const string& file_name) {
 
   /*--- Read header ---*/
   SkipToFlag(file_stream, line, "<Header>");
-
+  table_dim = 2;
+  bool found_level_count = false;
   while (GetNextNonEmptyLine(file_stream, line) && !eoHeader) {
 
-    /*--- number of points in LUT ---*/
+    /*--- check version_lut ---*/
     if (line.compare("[Version]") == 0) {
       GetNextNonEmptyLine(file_stream, line);
       version_lut = line;
+
+      if(version_lut.compare("1.0.1") == 0){
+        table_dim = 2;
+        n_levels = 1;
+        found_level_count = true;
+        n_points      = new unsigned long[n_levels];
+        n_triangles   = new unsigned long[n_levels];
+        n_hull_points = new unsigned long[n_levels];
```

Comment:
yes, in any place where you are using `new`, replace by std::vector or su2vector